src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,30 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import userFromServer from '../../api/users';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const currentUser = userFromServer.find(elem => elem.id == todo.userId);
+
+  if (!currentUser) {
+    return;
+  }
```

Comment:
Returning `undefined` from a React component can lead to rendering issues. Consider returning `null` instead to explicitly indicate that nothing should be rendered if `currentUser` is not found.